docs/snippets/ov_preprocessing_migration.c

```diff
@@ -0,0 +1,149 @@
+// Copyright (C) 2018-2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+#include <openvino/c/openvino.h>
+
+int main_new() {
+    char* model_path = NULL;
+    char* tensor_name = NULL;
+
+    ov_core_t* core = NULL;
+    ov_core_create(&core);
+    
+    ov_model_t* model = NULL;
+    ov_core_read_model(core, model_path, NULL, &model);
+
+    {
+    //! [ov_mean_scale]
+    ov_preprocess_prepostprocessor_t* preprocess = NULL;
+    ov_preprocess_input_info_t* input_info = NULL;
+    ov_preprocess_input_model_info_t* input_model = NULL;
+    ov_layout_t* layout = NULL;
+    ov_model_t* new_model = NULL;
+
+    ov_preprocess_prepostprocessor_create(model, &preprocess);
+    ov_preprocess_prepostprocessor_get_input_info_by_index(preprocess, 0, &input_info);
+    ov_preprocess_input_info_get_model_info(input_info, &input_model);
+    // we only need to know where is C dimension
+    ov_layout_create("...C", &layout);
+    ov_preprocess_input_model_info_set_layout(input_model, layout);
```

Comment:
In C++ version, here is:
`input.preprocess().mean(116.78f).scale({ 57.21f, 57.45f, 57.73f });`
But C 2.0 API doesn't provide mean() and scale() interface, do we need add such interfaces?

Reply:
As I see we also don't have:

- PreProcessSteps& reverse_channels();
- PreProcessSteps& convert_layout();
- PreProcessSteps& crop();
- PreProcessSteps& scale();
- PreProcessSteps& mean();

Honestly I am not sure that anybody will use it, but if we would like to have a full picture - we need to add all these methods. We already have:

- PreProcessSteps& convert_element_type();
- PreProcessSteps& convert_color();
- PreProcessSteps& resize();